icons/turtle.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m14 14-2-4H8l-2 4v3c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1v-3a8 8 0 0 1 16 0v3c0 .6-.4 1-1 1h-2c-.6 0-1-.4-1-1Z" />
+  <path d="M4.8 7.9 8 10" />
+  <path d="m12 10 3.2-2.1" />
+  <path d="M16.9 10H20a2 2 0 0 1 0 4H2" />
```

Comment:
Overoptimized again:
```suggestion
<path d="m12 10 2 4v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3a8 8 0 0 0-16 0v3a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-3l2-4h4Z" />
  <path d="M4.8 7.9 8 10" />
  <path d="m12 10 3.2-2.1" />
  <path d="M16.93 10H20a2 2 0 0 1 0 4H2" />
```

Reply:
Ah was just fixing the lines on it’s back… https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit can be a bit too keen! :)